shell/platform/windows/flutter_window_unittests.cc

```diff
@@ -400,5 +401,34 @@ TEST(FlutterWindowTest, InitialAccessibilityFeatures) {
   win32window.SendInitialAccessibilityFeatures();
 }
 
+// Ensure that announcing the alert propagates the message to the alert node.
+// Different screen readers use different properties for alerts.
+TEST(FlutterWindowTest, AlertNode) {
```

Comment:
Could you also add tests that the root node properly delegates to the window's `IAccessible`?

Reply:
Do you have any thoughts on how/where this test would be best implemented?

Reply:
Ideally we'd have integration tests that's spin up a Flutter engine and verify accessibility behaviors:

1. Send the announce platform message, check a mock `NotifyWinEvent` is called with the expected parameters
2. Somehow setup some accessibility nodes, send a `WM_GETOBJECT` message, and verify the native accessibility tree is expected.

I'm not sure how feasible this is. For the test I suggested above, I'd consider a new file since it seems Windows's current tests mostly targets the accessibility bridge.

Reply:
> Do we have existing unit tests that send platform messages? 

One was added just recently: https://github.com/flutter/engine/pull/36853

I'm still thinking on the rest of your comment, will loop back

Reply:
I think your latest tests are good, marking this as resolved